.config-schema.json

```diff
@@ -12,18 +12,30 @@
             "default": 100,
             "x-env-variable": "OPENFGA_MAX_TYPES_PER_AUTHORIZATION_MODEL"
         },
+        "maxConcurrentReads": {
+            "description": "The maximum allowed number of concurrent reads during queries.",
+            "type": "integer",
+            "default": 30,
+            "x-env-variable": "OPENFGA_MAX_CONCURRENT_READS"
+        },
         "changelogHorizonOffset": {
             "description": "The offset (in minutes) from the current time. Changes that occur after this offset will not be included in the response of ReadChanges.",
             "type": "integer",
             "default": 0,
             "x-env-variable": "OPENFGA_CHANGELOG_HORIZON_OFFSET"
         },
         "resolveNodeLimit": {
-            "description": "Defines how deeply nested an authorization model can be.",
+            "description": "Defines how deeply nested an authorization model can be before a Check request errors out.",
```

Comment:
🤔 Expand too? Where? I can't find where we throw an error in that case

Reply:
Sorry, not in Expand. Just in Check and ListObjects. But, in general, our resolution depth limitations apply to any endpoint that involves graph resolution, and so that's the intent behind the comment. It's not just specific to Check.